sensor-gateway/common/error/ErrorConstants.h

```diff
@@ -42,6 +42,8 @@ namespace ErrorHandling {
     namespace Message {
         std::string const SEPARATOR = " - ";
         std::string const EMPTY_MESSAGE = "";
+
+        std::string const PARAMETER_NOT_AVAILABLE = "Parameter is not available";
```

Comment:
Clang-Tidy: Initialization of 'PARAMETER_NOT_AVAILABLE' with static storage duration may throw an exception that cannot be caught

Warning Message for all const created in this file.